content/09.overfitting.md

```diff
@@ -8,6 +8,12 @@ To continue the student analogy, a smarter student has greater potential for mem
 
 ![A visual example of overfitting and failure to generalize. While a high-degree polynomial gets high accuracy on its training data, it performs poorly on data unlike that which it has seen before. In contrast, a simple linear regression works well on both datasets. The greater representational capacity of the polynomial is analogous to using a larger or deeper neural network.](images/overfitting.png){#fig:overfitting-fig}
 
+To evaluate deep supervised learning models, they should be trained, tuned, and tested on non-overlapping datasets.
+The data used for testing should be locked and only used one-time for evaluating the final model after all tuning steps are completed.
+Using a test set more than once will lead to biased estimates of the generalization performance [@arxiv:1811.12808; @doi:10.1162/089976698300017197].
+Additionally, many conventional metrics for classification (e.g. area under the receiver operating characteristic curve or AUROC) have limited utility in cases of extreme class imbalance [@pmid:25738806].
+Model performance should be evaluated with a carefully picked panel of relevant metrics that make minimal assumptions about the composition of the testing data [@doi:10.1021/acs.molpharmaceut.7b00578], with particular consideration given to metrics that are most directly applicable to the task at hand.
```

Comment:
Give specific example of relevant metrics?

Reply:
I don't want this to block merging so I've made a new issue #267 to track this since we should clarify